indigo_api/models.py

```diff
@@ -227,6 +229,10 @@ def repeal(self):
                 self._repeal = RepealEvent(self.repealed_date, self.repealed_by.title, self.repealed_by.frbr_uri)
         return self._repeal
 
+    @property
+    def place(self):
+        return self.locality if self.locality else self.country
```

Comment:
Nice. You can also do `return self.locality or self.country`, but no need to change it.